src/components/PeopleTableRow/PeopleTableRow.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { TableRowLink } from '../TableRowLink';
+
+interface Props {
+  person: Person
+}
+
+export const PeopleTableRow: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': person.slug === slug,
+      })}
+    >
+      <td>
+        <Link
+          to={person.slug}
+          className={classNames(
+            { 'has-text-danger': person.sex === 'f' },
+          )}
+        >
+          {person.name}
+        </Link>
```

Comment:
You already have a TableRowLink, I think, with very slight modifications, It can be used here as well?